README.md

```diff
@@ -1,25 +1,24 @@
-[![Build Status](https://travis-ci.org/tapanalyticstoolkit/spark-tk.svg?branch=master)](https://travis-ci.org/tapanalyticstoolkit/spark-tk)
+[![Build Status](https://travis-ci.org/trustedanalytics/spark-tk.svg?branch=master)](https://travis-ci.org/trustedanalytics/spark-tk)
```

Comment:
I have not seen a build status in a README.md file before. What does it do?
Also it points to trustedanalytics instead of tapanalyticstoolkit. Should that change?